src/main/scala/edu/ie3/simona/ontology/messages/services/WeatherMessage.scala

```diff
@@ -69,8 +69,8 @@ object WeatherMessage {
     *   Wind velocity
     */
   final case class WeatherData(
-      diffIrr: ComparableQuantity[Irradiance],
-      dirIrr: ComparableQuantity[Irradiance],
+      diffIrr: Irradiance,
+      dirIrr: Irradiance,
```

Comment:
Why are only the first two attribute types changed? Should we also change the temperature to ``squants``?

Reply:
Thanks for asking! The reason here is similar to the one above in `CylindricalThermalStorage`. For `Temperature` the challenge is in `calculateAirDensity` in `WecModel` since the Mass representation in squants have no method to divide through `MolarEnergy` which one gets if the universal gas constant is multiplied by the temperature. Since the speed effect we do not gain here is not that high I suggest to rather keep the quantities here as they are. But ideas are welcome here. Similar for WindSpeed at `determinePower` in `WecModel` for the cubed velocity.

Reply:
But you're right that these could and should be squants in the long term. So an idea could be to track this via an issue.